src/components/AlertBox/index.tsx

```diff
@@ -1,6 +1,14 @@
-import { Alert, AlertIcon, AlertProps, AlertStatus } from "@chakra-ui/react"
+import {
+  Alert,
+  AlertIcon,
+  AlertProps,
+  AlertStatus,
+  useColorMode,
+  Image,
+} from "@chakra-ui/react"
 import { CheckIcon, InfoIcon, WarningIcon } from "@chakra-ui/icons"
 import { FC } from "react"
+import magicAlertIcon from "../../static/images/MagicAlertIcon.png"
```

Comment:
Actually I just noticed that the build is giving this error:

```
[!] (plugin rpt2) Error: /home/runner/work/components/components/src/components/AlertBox/index.tsx(11,28): semantic error TS2307: Cannot find module '../../static/images/MagicAlertIcon.png' or its corresponding type declarations.
```

It has something to do with this line. Let's fix that before the merge

Reply:
@georgeweiler I managed to fix that in this PR - https://github.com/threshold-network/components/pull/17

Do a review, check if it's working and merge it to this branch and after that i will merge this branch into the `main`.